paper-api/src/main/java/org/bukkit/inventory/view/builder/LocationInventoryViewBuilder.java

```diff
@@ -0,0 +1,63 @@
+package org.bukkit.inventory.view.builder;
+
+import net.kyori.adventure.text.Component;
+import org.bukkit.Location;
+import org.bukkit.inventory.InventoryView;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * An InventoryViewBuilder that can be bound by location within the world
+ *
+ * @param <V> the type of InventoryView created from this builder
+ */
+@ApiStatus.Experimental
+public interface LocationInventoryViewBuilder<V extends InventoryView> extends InventoryViewBuilder<V> {
+
+    @NotNull
+    @Override
+    LocationInventoryViewBuilder<V> copy();
+
+    @NotNull
+    @Override
+    LocationInventoryViewBuilder<V> title(final @NotNull Component title);
```

Comment:
dont enforce titles

Reply:
Not sure what you mean by this but if you're suggesting I shouldn't enforce passing a title that's not possible.

Reply:
Yeah, the server doesn't really have a relationship between menutype and default titles. Figuring out how to expose those titles somehow is a different problem, plus all the translation keys are all public anyways, people can go look through them.

Reply:
If the title is not defined the client will use the default one
so I am in favor of nullable titles